integration/test_helpers/helpers.go

```diff
@@ -25,3 +37,17 @@ func RunCommand(cmd string, args ...string) *Session {
 
 	return session
 }
+
+func GetTargetURL() (string, error) {
```

Comment:
I wonder if the command should instead do the checks that URL is valid? I think it does, if not it should. Please check. Thx